Contribute/how-to-write-docs-auth-pack.md

```diff
@@ -129,25 +129,25 @@ The content team will review your template and merge the PR if it meets the styl
 
 Here's a short video that demonstrates the following features of the Learn Authoring Pack:
 
-* **YAML files**
-  * Support for "Learn: Link to file in repo"
-* **Markdown files**
-  * Update "ms.date" Metadata Value context menu option
-  * Code auto-completion support for code-fence language identifiers
-  * Unrecognized code-fence language identifier warnings / auto correction support
-  * Sort selection ascending (A to Z)
-  * Sort selection descending (Z to A)
+- **YAML files**
+  - Support for "Learn: Link to file in repo"
+- **Markdown files**
+  - Update "ms.date" Metadata Value context menu option
+  - Code auto-completion support for code-fence language identifiers
+  - Unrecognized code-fence language identifier warnings / auto correction support
+  - Sort selection ascending (A to Z)
+  - Sort selection descending (Z to A)
 
-> [!VIDEO https://www.youtube.com/embed/6zfbBRdjlw8]
+> [!VIDEO <https://www.youtube.com/embed/6zfbBRdjlw8>]
```

Comment:
```suggestion
> [!VIDEO https://www.youtube.com/embed/6zfbBRdjlw8]
```

Reply:
Oh that was unintended. Must have been the Learn Author Pack that did it on a general formatting.